dm/loader/lightning.go

```diff
@@ -16,27 +16,29 @@ package loader
 import (
 	"context"
 	"path/filepath"
+	"strings"
 	"sync"
 
-	"github.com/docker/go-units"
+	"github.com/pingcap/ticdc/dm/dm/config"
+	"github.com/pingcap/ticdc/dm/dm/pb"
+	"github.com/pingcap/ticdc/dm/dm/unit"
+	"github.com/pingcap/ticdc/dm/pkg/binlog"
+	"github.com/pingcap/ticdc/dm/pkg/conn"
+	tcontext "github.com/pingcap/ticdc/dm/pkg/context"
+	"github.com/pingcap/ticdc/dm/pkg/log"
+	"github.com/pingcap/ticdc/dm/pkg/utils"
+
 	"github.com/pingcap/failpoint"
 	"github.com/pingcap/tidb-tools/pkg/dbutil"
 	"github.com/pingcap/tidb/br/pkg/lightning"
 	"github.com/pingcap/tidb/br/pkg/lightning/common"
 	lcfg "github.com/pingcap/tidb/br/pkg/lightning/config"
 	"github.com/pingcap/tidb/parser/mysql"
+
+	"github.com/docker/go-units"
 	"go.etcd.io/etcd/clientv3"
 	"go.uber.org/atomic"
 	"go.uber.org/zap"
-
-	"github.com/pingcap/ticdc/dm/dm/config"
-	"github.com/pingcap/ticdc/dm/dm/pb"
-	"github.com/pingcap/ticdc/dm/dm/unit"
-	"github.com/pingcap/ticdc/dm/pkg/binlog"
```

Comment:
i am not sure if this import ordering is wrong...

some file is 
```
1. standard library,
2. project packages
3. third party packages
```

bu some file like [dm-master](https://github.com/pingcap/ticdc/blob/61c7a4e77676bb6110a15d2377285baca5b8fb98/dm/dm/master/etcd.go#L30) is 
```
1. standard library,
2. third party packages
2. project packages
```

there is a issue in golang repo releatd to this https://github.com/golang/go/issues/20818, 
maybe we should keep the group ordering in file as before ?🤔

Reply:
Done